### ISSUE TITLE
Avoid logging error while log-level is warn

### ISSUE THREAD

**Issue by iedmrc** [TheAppleFreak/winston-slack-webhook-transport#12]
Hi,
I need to send warning messages but not error messages to a Slack channel. How can/should I achieve this? I tried to send `false` in formatter (as suggested here: https://github.com/winstonjs/winston#filtering-info-objects) but it throws:
```
(node:11417) UnhandledPromiseRejectionWarning: Error: Request failed with status code 400
```

Thanks!

**Comment by TheAppleFreak** [comment]
Hmm, that's interesting. I'll check to see if there's something that this module needs to do that I'm not doing. Sorry for the inconvenience!

**Comment by daxadal** [comment]
I encountered the same casuistry, so I suggest a custom solution. The following functionalities have been added:
- SlackHookOptions formatter can return false to filter out messages
- SlackHookOptions formatter types updated
- New test configuring SlackHook with a custom formatter that changes the output
- New test configuring SlackHook with a custom formatter that filters out the message

Pull request: https://github.com/TheAppleFreak/winston-slack-webhook-transport/pull/14

**Comment by TheAppleFreak** [comment]
@_@ 

Sorry about the delay, life got weird for me! Merged #14, so this should be resolved now.
